server/models/Notification.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require("mongoose");
+
+const notificationSchema = new mongoose.Schema({
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: 'User'
+  },
+  type: {
+    type: String,
+    enum: ['request', 'payment', 'message'],
```

Comment:
```suggestion
    enum: ['request', 'payment', 'message'],
    lowercase: true,
```
We can specify additional properties for a path. For example, if you want to lowercase a string before saving